src/bthread/task_control.h

```diff
@@ -103,8 +103,7 @@ friend bthread_t init_for_pthread_stack_trace();
 
 private:
     typedef std::array<TaskGroup*, BTHREAD_MAX_CONCURRENCY> TaggedGroups;
-    static const int PARKING_LOT_NUM = 4;
-    typedef std::array<ParkingLot, PARKING_LOT_NUM> TaggedParkingLot;
+    typedef std::array<ParkingLot, BTHREAD_MAX_CONCURRENCY> TaggedParkingLot;
```

Comment:
Can define a new constant BTHREAD_MAX_PARKINGLOT and add validator for bthread_parking_lot_of_each_tag ?

Reply:
done